src/pages/contact.mdx

```diff
@@ -26,18 +26,31 @@ San Francisco
 
 #### Or leave us a msg :)
 
-<form
-  action="https://formspree.io/anand@codebrahma.com"
-  method="post" target='/thanks'>
-    <Flex flexWrap='wrap' flexDirection='column'>
-      <Box width={[1,1/3]}>
-        <InputText size='25' rows="5" style={{width:'100%'}} required as='textarea' name='msg' placeholder='Tell us about your idea'/>
-      </Box>
-      <Box width={[1,1/3]} mt={1}>
-        <InputText style={{width:'100%'}} name='email' type='email' required placeholder='Email address'/>
-      </Box>
-      <Box width={[1,1/3]} mt={1}>
-        <InputButton/>
-      </Box>
-    </Flex>
+<form id="contact-form">
+  <Flex flexWrap='wrap' flexDirection='column'>
+    <Box width={[1,1/3]}>
+      <InputText
+        id="idea" 
+        size='25'
+        rows="5"
+        style={{width:'100%'}}
```

Comment:
wont it take `width={1}` prop

Reply:
@Kameshwaran `width={1}` won't cover the whole Box, maybe that's the reason Anand resorted to a `style` prop